examples/avm/buildExportTx-PChain.ts

```diff
@@ -48,7 +49,7 @@ const main = async (): Promise<any> => {
     avaxAssetID
   )
   const balance: BN = new BN(getBalanceResponse.balance)
-  const amount: BN = balance.sub(fee)
+  const amount: BN = balance.sub(fee).sub(ONEAVAX)
```

Comment:
In my testing I swapped C->P->X->C->X->P->C. This tests each combination of export/import between the 3 chains C/P/X.

These `example/` scripts transfer the full balance of the `ewoq` address in each swap. I subtracted 1 AVAX so that the `ewoq` address would have enough funds to pay for the `ImportTx` in my C->P->X->C->X->P->C test workflow.

[I've remove the 1 AVAX subtraction from each example script](https://github.com/ava-labs/avalanchejs/commit/ae9c5b981fc5e9574336218eff63d1b043e79c8b)